src/hooks/lib/apr/useTokenGlpVault.ts

```diff
@@ -26,13 +24,45 @@ export default function useTokenGlpVault({
     enabled: enabled,
   })
 
-  const ethRewardsPerSecond = glpQuery.data
-
-  const fortGlpAprFallback = useQuery([chainId, asset, "fortGlpAprFallback"], {
-    queryFn: () => getFortGlpAprFallback(ethRewardsPerSecond),
-    retry: false,
-    enabled: enabled && !!ethRewardsPerSecond,
+  const result = useGetFortGlpAprFallback({
+    ethRewardsPerSecond: glpQuery.data,
+    enabled: enabled && !!glpQuery.data,
   })
 
-  return fortGlpAprFallback
+  return {
+    isLoading: false,
+    data: result,
+  }
+}
+
+export function useGetFortGlpAprFallback({
+  ethRewardsPerSecond,
+  enabled,
+}: {
+  ethRewardsPerSecond?: bigint
+  enabled?: boolean
+}) {
+  const [ethPrice, setEthPrice] = useState(0)
+  useEffect(() => {
+    async function fetchPrice() {
+      const price = await getApiPrice({ asset: ethTokenAddress })
+      setEthPrice(price)
+    }
+    fetchPrice()
+  }, [])
+
+  const { aumInUsdg: aum, price: priceGmx } = useGlpPrice({ enabled })
+  const ethRewardsAnnual =
+    (Number(ethRewardsPerSecond ?? 0n) * 3600 * 24 * 365) / 1e18
```

Comment:
This seems like it could explode easily if `ethRewardsPerSecond` was a large enough value.

Would this work?

```ts
export function useGetFortGlpAprFallback({
  ethRewardsPerSecond = 0n,
  enabled
}) {
  // ...
  const ethRewardsAnnual = Number((ethRewardsPerSecond * 3600n * 24n * 364n / 1e18n)
```

Reply:
Or alternatively use like, formatUnits or something.